sdk/remoterendering/azure-mixedreality-remoterendering/pom.xml

```diff
@@ -52,6 +52,16 @@
       <version>1.11.19</version> <!-- {x-version-update;com.azure:azure-core-http-okhttp;dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-http-jdk-httpclient</artifactId>
+      <version>1.0.0-beta.11</version> <!-- {x-version-update;com.azure:azure-core-http-jdk-httpclient;dependency} -->
+    </dependency>
```

Comment:
To have the tests work properly with Java 8, you'll want to remove this dependency and leave what is added later on with the JDK 12+ profile

Reply:
<profiles>
    <profile>
      <id>java12plus</id>
      <activation>
        <jdk>[12,)</jdk>
      </activation>
      <dependencies>
        <dependency>
          <groupId>com.azure</groupId>
          <artifactId>azure-core-http-jdk-httpclient</artifactId>
          <version>1.0.0-beta.11</version> <!-- {x-version-update;com.azure:azure-core-http-jdk-httpclient;dependency} -->
          <scope>test</scope>
        </dependency>
      </dependencies>
    </profile>
  </profiles>